misc/vtk/include/inviwo/vtk/util/lambda2func.h

```diff
@@ -0,0 +1,35 @@
+#ifndef IVW_LAMBDA2FUNC_H
+#define IVW_LAMBDA2FUNC_H
+
+#include <vtkObject.h>
+#include <type_traits>
+#include <utility>
+
+namespace inviwo {
+template <typename Callable>
+union storage {
+    storage() {}
+    ~storage() {}
+    std::decay_t<Callable> callable;
+};
+
+template <int, typename Callable, typename Ret, typename... Args>
+auto fnptr_(Callable&& c, Ret (*)(Args...)) {
```

Comment:
what is this? why don't we just used std::function

Reply:
Used in https://github.com/inviwo/modules/blob/master/tensorvis/tensorvisio/src/processors/vtkreader.cpp and https://github.com/inviwo/modules/blob/master/tensorvis/tensorvisio/src/processors/vtkunstructuredgridtorectilineargrid.cpp 
I just tried to change from fnptr to std::function but then it does not compile

Reply:
```
Error	C2664	 'void vtkCallbackCommand::SetCallback(void (__cdecl *)(vtkObject *,unsigned long,void *,void *))': cannot convert argument 1 from 'std::function<void (vtkObject *,unsigned long,void *,void *)>' to 'void (__cdecl *)(vtkObject *,unsigned long,void *,void *)'	inviwo-module-tensorvisio	E:\github-master-repo\inviwo\additional\modules\tensorvis\tensorvisio\src\processors\vtkreader.cpp	161	
 ```